hack/scorecard.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Copyright 2020 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+echo "Cleaning up CRDs to run the scorecard tool"
+for crd in deploy/crds/*_crd.yaml; do
+  [[ -e "$crd" ]] || break
+  oc delete -f "$crd"
```

Comment:
Don't we need to reapply CRDs afterwards?

Reply:
The scorecard is run on a test/temp env. so won't matter. But I think won't cause a harm having them replied.